css/contact.css

```diff
@@ -0,0 +1,142 @@
+/*
+===================================
+===================================
+Styles for an individual page
+
+- Rename this file to match the page you are working on (e.g. home.css, about.css, etc.)
+- Create a new file for your page and link this file to it in the <head> of your HTML document
+- Variables from global.css will work in this CSS file as well. Do not add any variables to this file.
+===================================
+===================================
+*/
+/* 
+===================================
+Page Component Styles (icon grid, team member card)
+===================================
+*/
+
+/* contact form */
+.wrapper {
+	width: var(--mobile-wrapper-width);
+}
+.form-input {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-around;
+	/* padding: 20px; */
+}
+input[type="text"],
+[type="email"],
+select,
+textarea {
+	resize: vertical;
+	font-family: var(--main-font);
+	font-size: var(--font16);
+	font-weight: var(--font-weight-regular);
+	color: var(--color-onyx);
+	background-color: var(--color-white);
+	border: 1px solid #6c6c6c;
```

Comment:
Use variables where possible from here down.

Reply:
the 6c6c6c does not exist in the global styles